### PR TITLE
HCIDOCS-125: Updating AI docs location in OCP docs.

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: [HCIDOCS-125](https://issues.redhat.com/browse/HCIDOCS-125)

For: Version 4.14

Preview [here](https://71277--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno)

Signed-off-by: Katie Drake kdrake@redhat.com

### CODE REVIEW

File: installing/installing_sno/install-sno-installing-sno.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
+See link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/[Assisted Installer for OpenShift Container Platform] to install {product-title} on a single node. You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
```
Comment (apinnick) [inline installing/installing_sno/install-sno-installing-sno.adoc:9]: I don't think this link is visible enough. I recommend adding it after the header:

== Installing {sno} using the Assisted Installer

To install {product-title} on a single node, use the web-based Assisted Installer wizard to guide you through the process and manage the installation.

See the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/[Assisted Installer for OpenShift Container Platform]  documentation for details and configuration options.

Comment (apinnick) [reply installing/installing_sno/install-sno-installing-sno.adoc:9]: Also you should probably use the product-title attribute here.

Comment (kdrake20) [reply installing/installing_sno/install-sno-installing-sno.adoc]: Updated As Suggested.

**Comment by kdrake20** [comment]
**Timestamp:** Wednesday, February 7, 2024 10:47 AM

**Summary:** @apinnick, I've made your requested changes. Could you please review and provide an `/lgtm` or updated feedback? Thank you!

**Next Steps:**
- [ ]  Receive `/lgtm` from Avital

**Comment by kdrake20** [comment]
**Timestamp:** Thursday, February 8, 2024 1:31 PM
**Summary:** Added `/label peer-review-needed1 to request peer review.

**Next Steps:**
- [ ] Receive `/lgtm` from peer review
- [ ] Request merge review

**Comment by tmalove** [comment]
@kdrake20  don't forget to squash your commits before the merge request...otherwise, looks great!
/remove-label peer-review-in-progress
/label peer-review-done
/LGTM

**Pull request merged**